client/v3/experimental/recipes/double_barrier.go

```diff
@@ -71,10 +71,9 @@ func (b *DoubleBarrier) Enter() error {
 			// delete itself now, otherwise other processes may need to wait
 			// until these keys are automatically deleted when the related
 			// lease expires.
-			if err = b.myKey.Delete(); err != nil {
-				// Nothing to do here. We have to wait for the key to be
-				// deleted when the lease expires.
-			}
+			b.myKey.Delete()
+			// Nothing to do here even if we run into error.
+			// We have to wait for the key to be deleted when the lease expires.
```

Comment:
Suggest to keep this unchanged. Previously code explicitly commenting why the error is ignored. 

It can't even pass the static check, because all returned errors should be explicitly ignored (i.e. _ = xxxx) or be checked. The reason why all the workflows are green is probably because some checks/lints are disabled?

Reply:
> It can't even pass the static check, because all returned errors should be explicitly ignored (i.e. _ = xxxx) or be checked. The reason why all the workflows are green is probably because some checks/lints are disabled?

we don't enable `errcheck` before. So it can pass the check.

> Suggest to keep this unchanged. Previous code explicitly commenting why the error is ignored.
Sounds good. I was thinking that it's good enough to keep the comment. I updated with disable linter comment. Please take a look.